### PR TITLE
[doc] Add instructions to setup an interconnect (enable building) to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by bgeuken** [comment]
@lethliel ^

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -92,20 +92,28 @@ We are using [Vagrant](https://www.vagrantup.com/) to create our development env
 You can access the frontend at [localhost:3000](http://localhost:3000). Whatever you change in your cloned repository will have effect in the development environment.
 **Note**: The vagrant instance is configured with a default user 'Admin' and password 'buildservice'.
 
-9. Changed something in the frontend? Test your changes!
+9. Building packages:
+     To be able to build packages, you need to:
```
Comment (adrianschroeter) [inline CONTRIBUTING.md]: An interconnect is not always needed and might push people away if they have security concerns. I would write it like: 

The easiest way to start building is to create an interconnect to build.opensuse.org. All resources, including the base distributions can be used that way directly.

Comment (bgeuken) [inline CONTRIBUTING.md]: Good point. I've updated it.

**Comment by lethliel** [comment]
@bgeuken Awesome. Thanks and LGTM

**Pull request merged**